content/basic-use.md

```diff
@@ -0,0 +1,182 @@
+
+In their day to day work, most Ember developers use only a small number of CLI commands.
+We'll cover them here, along with a quick tutorial of how to use the `--help` option.
+
+## Using the help command
+
+For any of the commands below, you can see all of the options available by using the `--help` flag.
+
+For example, `ember generate --help` will show you a full list of all the types of files you can create using the CLI.
+
+<!-- SCREENSHOT -->
+
+## Create an app
+
+### Format:
+
+```
+ember new <my-app-name> [options]
+```
+
+### What it does
+
+`ember new` creates all the boilerplate files that are part of an Ember app. It puts them in a folder with the same name as whatever we provide in place of `<my-app-name>`.
+
+### Example use
+
+This command below will create a folder called `camping-trip-tracker`, which will be full of app files. It uses the `--yarn` option to show that the app should use `yarn` by default. Yarn is a package manager alternative to `npm`. Options typically start with a double dash `--` and can be omitted entirely.
+
+```
+ember new camping-trip-tracker --yarn
+```
+
+### Learn more
+
+<!-- Link to quickstart in the guides -->
+
+## Serve the app locally
+
+### Format
+
+```
+ember serve [options]
+```
+
+You can also use shorthand like `ember s`.
+
+### What it does
+
+`ember serve` takes all of the app's files and turns them into something that can be rendered in the browser. By default, view the app by visiting `http://localhost:4200`. It's a good idea to keep the server running as we work so that we know as soon as we've broken something. The CLI watches the project folders, and will rerender as files change.
+
+### Example
+
+By default, apps are served at port `4200`, but if you need to change it for some reason, you could visit your app at `http://localhost:3200` by using this command:
+
+```
+ember serve --port 3200
+```
+
+### Learn more
+
+<!-- link to quickstart in the guides -->
+
+## Generate app files
+
+### Format
+
+`ember generate <type of file> <the name to give it>`
+
+### What it does
+
+`ember generate` creates new files within your app. For example, you can use it to create components, routes, services, models, and more. For a full list, type `ember generate --help`. 
+
+The new files will contain the necessary boilerplate, they will go in the right place, and the CLI will make sure that file naming conventions are followed. For example, components must always have a dash in their names. Creating files by hand is not recommended because mistakes can lead to confusing error messages.
+
+### Example
+
+This command will make a component named `packing-list`. There will be three files created in the app: `packing-list.hbs` which will define what it looks like, `packing-list.js` with JavaScript code to handle user interaction, and an integration test file called `packing-list-test.js`.
+
+```
+ember generate component packing-list
+```
+
+### Learn more
+
+<!-- link to custom blueprints -->
```

Comment:
https://ember-cli.com/generators-and-blueprints
https://ember-cli.com/extending/